app/templates/join.hbs

```diff
@@ -1,16 +1,17 @@
 {{page-title 'Join'}}
-<section class="join__container">
-{{#if this.login.isLoading}}
-<div data-test-loading class='intro__loading'>
-    <Fa-Icon @size='2x' @icon='circle-notch' @spin={{true}} />
-</div>
-{{else}}
-{{#if this.login.userData.roles.archived}}
-<OnboardingCard>
-<h3 class="archived__heading">Archived User</h3>
-</OnboardingCard>
-{{else}}
- <Stepper />
-{{/if}}
-{{/if}}
+<section class='join__container'>
+  {{#if this.login.isLoading}}
+    <div data-test-loading class='intro__loading'>
+      <Fa-Icon @size='2x' @icon='circle-notch' @spin={{true}} />
+    </div>
+  {{else}}
+    {{#if this.login.userData.roles.archived}}
+      <OnboardingCard>
+        <h3 class='archived__heading'>Archived User</h3>
```

Comment:
why is the class name named so specific?
What's wrong with something like only `heading`

Tomorrow If I decide to rename the title to something else, then the classname will not make sense here